docs/api/data.md

```diff
@@ -7,6 +7,244 @@ import WorkInProgress from '../components/WorkInProgress.mdx'
 
 # Data API
 
-A rich graphql based data API that can be used to query for any file metadata based on various query parameters.
+Data API provides a graphql based interface for storing and querying structured data. Decentralized application developers for Box can use this API to create, update and delete JSON documents using a standard graphql interface directly on their Box. The Data API is a part of [Borg Client](https://github.com/functionland/fula/tree/main/libraries/borg-client) and you can use it using the `graphql` interface.
 
+Currently, there are 4 types of mutation and a single query type that you can use. In this document you can find the definition and a simple example for each operation, if you are familiar with graphql schemas, you may find the current [graphql schema](https://github.com/functionland/fula/blob/main/apps/box/src/graph/gql-engine/schema.ts)  for the Data API useful.
+
+
+## Queries
+Every query operation takes a `collection` argument that refers to the name for collection you want to query. If a collection with this name does not exist, the operation simply returns an empty output.   
+### read
+Fetches a previously stored document based on a filter object.
+
+`read (input:ReadInput): [JSON]`
+
+The `input` argument should contain:
+- `collection:String` (required): name of the collection.
+- `filter: JSON`: a [Filter](#Filter Objects) object that determines each document's existence in the output.
+
+#### Example
+This query operation finds all `profile` documents that has an `age` field greater than `20` and then returns their `id, name, age`.  
+
```

Comment:
If this is React it would be great if there were a page somewhere describing how to get this code working in my own app.